config/eslintrc_react.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         'react/default-props-match-prop-types': 1,
         'react/display-name': 0, // Auto covered by jsx transformer.
         'react/forbid-component-props': 1,
+        'react/forbid-dom-props': 0,
```

Comment:
👍 

In my _weak_ opinion, it's more better to forbid `id` attribute for plain DOM element.  It's difficult to ensure that the`id` value  is _unique_ in the document because React's component model encapsulate and hide its children and user should not know about the details of children components.

How do you think about this?

Reply:
あーたしかに。
plain DOM でidつかえなくなると不便じゃないっておもってたけど
`.jsx` 内ってことを忘れてた
そう考えるとおっしゃる通りで再利用性の高いコンポーネントにおいて
`id` が重複する可能性が高くなるのはガードしておきたいですね
なおしますmm